docs/_diagram_src/readme.txt

```diff
@@ -1,3 +1,40 @@
 Diagram Sources:
```

Comment:
I think you accidently left the _diagram_src folder there seems we are only using the diagrams folder now and you have the same readme.md in the diagrams folder.

Reply:
explanation in the commit message

Reply:
Do you have a ticket # for the removal of deprecated _diagram_src folder?

Reply:
no